catalog/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+from django.urls import path
+from . import views
+
+app_name = 'catalog'
+
+
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("home/", views.home_catalog, name="home"),
```

Comment:
Давай определим главную страницу так, чтобы на нее можно было перейти сразу после открытия сайта
```suggestion
    path("", views.home_catalog, name="home"),
```
А настройку админки перенести нужно в главный файл `config/urls.py`